src/main/webapp/game.js

```diff
@@ -31,44 +75,41 @@ async function addToChat() {
   });
 }
 
-function updateChat(data) {
-  let newChatItem = createChatItem(data);
-  let chatbox = document.getElementById("chatbox");
-  chatbox.insertAdjacentHTML("beforeend", newChatItem);
-  // Autoscroll to bottom on chat update
-  let elem = document.getElementById("chatbox");
-  elem.scrollTop = elem.scrollHeight;
-}
+document.onkeypress = function(e) {
+  if (e.key === "Enter") { //checks whether the pressed key is "Enter"
+    addToChat();
+  }
+};
 
-function createChatItem(data) {
-  let message = data.message;
-  let username = data.username;
-  let messageType = CSS_MESSAGE_CLASS_DICT[data.messageType];
-  return `<p class="${messageType}"><span class="username">${username}: </span>${message}</p>`;
+function embedVideo() {
+  fetch('/round').then(response => response.json()).then((roundMap) => {
+    videoId = roundMap.videoId;
+    startTime = roundMap.startTime;
+    endTime = roundMap.endTime;
+    document.getElementById("player").src = "https://www.youtube.com/embed/" + videoId;
+  });
 }
 
-// Connect Pusher
-Pusher.logToConsole = false;
-
-var pusher = new Pusher(CLIENT_KEY, {
-  cluster: "us2",
-});
+function createTimer() {
+  Timer = setInterval("setTimer()",ONE_SECOND);
+}
 
-var channel = pusher.subscribe(PUSHER_APPLICATION_NAME);
-channel.bind(PUSHER_CHAT_CHANNEL_NAME, function(data) {
-  updateChat(data);
-});
+function setTimer() {
+  let now = new Date().getTime();
+  if (startTime > 0 && now >= startTime) {
+    document.getElementById("timer").innerHTML = (Math.floor((endTime - now) / ONE_SECOND)) + "s";
```

Comment:
Might be a good idea to create a variable for document.getElementById("timer"), as its used twice and would make the code more readable